projects/sw-offline-google-analytics/README.md

```diff
@@ -32,8 +40,15 @@ and retried the next time the service worker starts up.
 // First, import the library into the service worker global scope:
 importScripts('path/to/offline-google-analytics-import.js');
 
-// Then, call goog.useOfflineGoogleAnalytics() to activate the library.:
-goog.useOfflineGoogleAnalytics();
+// Then, call goog.offlineGoogleAnalytics.initialize():
+goog.offlineGoogleAnalytics.initialize({
+  parameterOverrides: {
+    // Optionally, pass in an Object with additional parameters that will be
+    // included in each replayed request.
+    dimension1: 'Some Value',
+    dimension2: 'Some Other Value'
```

Comment:
`dimensionXX` is the [`analytics.js` field](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference) representation of custom dimension parameters, but since this is the raw hit payload, it should use the [Measurement Protocol parameters](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#customs), which are `cd1` and `cd2`.

It should probably also link to the parameter reference:
https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters